egs2/TEMPLATE/asr1/db.sh

```diff
@@ -20,6 +20,7 @@ if [[ "$(hostname -d)" == clsp.jhu.edu ]]; then
     HKUST1=
     HKUST2=
     LIBRISPEECH=
+    LJSPEECH=
```

Comment:
I think it’s better to move `db.sh` under the `TEMPLATES directory (or some common directory for both asr and tts recipes) or prepare it in both `asr1` and `tts1`.

Reply:
What does it mean?
You mean we should prepare two db.sh for tts and asr?

Reply:
sorry for the confusion. please ignore my comment.